source/content/guides/drupal/drupal-hosted-createempty-md/05-composer-upstream.md

```diff
@@ -74,8 +74,6 @@ This is a common location for the `config` file. If this isn't where your `confi
 
 ```bash{promptUser:user}
 git checkout main config
-git mv config/* config
-git rm -f web/sites/default/config/.htaccess
```

Comment:
Do we want to remove Line 78 or replace it with

```markdown
git rm -f config/.htaccess
```

Reply:
.htaccess doesn't affect Pantheon behavior. Some sniffers check to see if it's there and complain if it isn't. Might as well leave it there.